proto/event.proto

```diff
@@ -38,13 +38,12 @@ message Metric {
 message Counter {
   string name = 1;
   float val = 2;
-  float sampling = 3;
 }
 
 message Timer {
   string name = 1;
   float val = 2;
-  float sampling = 3;
+  float count = 3;
```

Comment:
Might be better off using an unsigned integer for `count`.

Reply:
Agree. My idea was to generalise for counting _multiples_ of things (like 0.125 thousands) but that's just crazy and nobody's really doing that.